scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -84,7 +84,7 @@ else
   if doIsMacOs
   then
     echo "Doing workarounds for MacOS quirks..."
-    echo -e "#!/bin/bash\n${DEVON_IDE_HOME}/software/eclipse/Eclipse.app/Contents/MacOS/eclipse \$@" > "${ECLIPSE_HOME}/eclipse"
+    echo -e "#!/bin/bash\n${DEVON_IDE_HOME}/software/eclipse/Eclipse.app/Contents/MacOS/eclipse "'"$@"' > "${ECLIPSE_HOME}/eclipse"
```

Comment:
I also think this introduces a bug. We are writing a script here and inside the generated script the parameters to eclipse should be `$@`. Anyhow I justed tested this and both versions are not working as exclamation mark needs to be escaped as it seems... But maybe also the latter differes from interactive shell and from script as this used to work...

Reply:
The change is right - if you ever intend to support spaces in options. $@ is an array variable, if it is used without double quotes spaces within different elements will split the option into more than one. If used WITH double quotes the shell will automatically protect those elements by double quotes:   1foo 2bar "3foo bar"

With respect to "!" - indeed, during interactive use this needs to be escaped when in double quotes. It would be better, to do this also within scripts. Anyway I personally would use a HERE document here instead of embedding \n within the string. 
In theory there could be a bash on some strange OS, that only accepts different line endings than a newline. (No, not bash on windows)